src/init.cpp

```diff
@@ -515,6 +515,12 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-peginconfirmationdepth", strprintf(_("Pegin claims must be this deep to be considered valid. (default: %d)"), DEFAULT_PEGIN_CONFIRMATION_DEPTH));
     }
     strUsage += HelpMessageOpt("-validatepegin", strprintf(_("Validate pegin claims. All functionaries must run this. (default: %u)"), DEFAULT_VALIDATE_PEGIN));
+    strUsage += HelpMessageOpt("-mainchainrpchost=<addr>", strprintf("The address which the daemon will try to connect to validate peg-ins, if enabled. (default: cookie auth)"));
+    strUsage += HelpMessageOpt("-mainchainrpcport=<port>", strprintf("The port which the daemon will try to connect to validate peg-ins, if enabled. (default: cookie auth)"));
+    strUsage += HelpMessageOpt("-mainchainrpcuser=<username>", strprintf("The rpc username that the daemon will use to connect to validate peg-ins, if enabled. (default: cookie auth)"));
+    strUsage += HelpMessageOpt("-mainchainrpcpassword=<password>", strprintf("The rpc password which the daemon will use to connect to validate peg-ins, if enabled. (default: cookie auth)"));
+    strUsage += HelpMessageOpt("-mainchainrpccookiefile=<path>", strprintf("The bitcoind cookie auth path which the daemon will use to connect to validate peg-ins, if enabled. (default: default bitcoind datadir)"));
```

Comment:
Please, s/(default: default bitcoind datadir)"/(default: %s)", COOKIEAUTH_FILE/ 
In fact, I would separate MAINCHAIN_COOKIEAUTH_FILE from COOKIEAUTH_FILE for extensibility and headackelessness(TM). See https://github.com/ElementsProject/elements/pull/286

I would ask the same for the rest of the parameters (ie print the actual default).

Reply:
Right, but for the other options the default is "uses cookie auth", not a variable.

Reply:
No, the default for the -mainchainrpccookiefile command line argument (which is the default being documented here) is currently COOKIEAUTH_FILE. See #286 .

Reply:
to be clear, the default would be `datadir/networkdir/COOKIEAUTH_FILE`.

I don't care a ton about this to be honest, and what I wrote maximally mimics the line:
> strUsage += HelpMessageOpt("-rpccookiefile=<loc>", _("Location of the auth cookie (default: data dir)"));

If you feel very strongly I can have it print out the whole thing for clarity, and might also be worthwhile upstream.

Reply:
no, if COOKIEAUTH_FILE is a complete path, that's the default on its own.
if it's not complete (which is the case), then the default is datadir/regtest/COOKIEAUTH_FILE for elementsregtest (the default chain), only after trying all that is the default datadir/networkdir/COOKIEAUTH_FILE. But describing that default is too complex.
The problem is actually that the argument has such complex behaviour more than the documentation. Never mind, we can solve this later, perhaps as part of #286

Reply:
good idea